common/components/ui/Modal.scss

```diff
@@ -1,132 +1,130 @@
-@import "common/sass/variables";
-@import "common/sass/mixins";
+@import 'common/sass/variables';
+@import 'common/sass/mixins';
 
 $m-background: #fff;
 $m-window-padding: 20px;
 $m-header-padding: 15px;
 $m-header-height: 62px;
 $m-content-padding: 20px;
-$m-footer-height: 58px;
 $m-close-size: 26px;
 $m-anim-speed: 400ms;
 
 @keyframes modalshade-open {
-  0% {
-    opacity: 0;
-  }
-  70%,
-  100% {
-    opacity: 1;
-  }
+	0% {
+		opacity: 0;
+	}
+	70%,
+	100% {
+		opacity: 1;
+	}
```

Comment:
Looks like your spaces became tabs in here, mind fixing your editor to keep them the same?